src/components/body/scroller.component.ts

```diff
@@ -69,7 +69,7 @@ export class ScrollerComponent implements OnInit, OnDestroy {
     requestAnimationFrame(() => {
```

Comment:
@hoffination I know this `requestAnimationFrame` was here before, but I'm curious as to why?  It might be more advantageous to debounce this but it seems out of scope for the PR.  I wonder if the authors had an intention that isn't obvious to me?

Reply:
Agreed. I'm not sure of the benefit

Reply:
See: https://www.html5rocks.com/en/tutorials/speed/rendering/. The rows are positioned using css animations.

Reply:
That makes sense, I usually debounce and I wondered if it was a vsync thing.